code/src/ssd1306.cpp

```diff
@@ -0,0 +1,46 @@
+#include "ssd1306.hpp"
+
+// ssd1306_i2c_c::
+namespace r2d2::display {
+    ssd1306_i2c_c::ssd1306_i2c_c(r2d2::i2c::i2c_bus_c &bus,
+                                 const uint8_t &address)
+        : bus(bus), address(address), cursor(255, 255) {
+        // wait for the controller to be ready for the initialization
+        hwlib::wait_ms(20);
```

Comment:
Is this the right place for a wait? This doesn't seem related to the display itself, and the `wait_ms` call has been changed  earlier this PR.

Reply:
fixed in #12